### DISCUSSION TITLE
How to authenticate the JWT sent from the Microsoft Teams MessageCard HttpPost

### DISCUSSION THREAD

**Discussion by ToastyCeral** [MicrosoftDocs/msteams-docs#10203] [Announcements]
Using Microsoft Teams MessageCard, I am able to utilize the HttpPost to send data to a ASP.Net Core service, but how do I authenticate the JWT with Azure Active Directory that is sent in the Authorization header? I know the JWT contains the logged-in AD user object id, but I cannot find any documentation on what to do once I have the value(s).

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/46486579/b2228965-54c6-4f0d-8778-7c4499658b3c)

**Comment by Wajeed-msft** [comment]
Raise an issue here: https://github.com/MicrosoftDocs/msteams-docs/issues/10835
